tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
In new projects, I like adding `"types"` to the config to specify what `@types` packages to load for global types.  (Otherwise all `@types` packages can contribute global types, even ones that aren't direct project dependencies)  In this case it'd probably be `"types": ["jasmine"]`.  

Looks like that might require some `lib` config; it starts complaining about stuff like `Object.assign` it looks like maybe some dependency is adding those types implicitly?